src/content/operations/troubleshooting/_index.en.md

```diff
@@ -25,19 +25,29 @@ Install subctl:
 curl -Ls https://get.submariner.io | VERSION=<your Submariner version> bash
 ```
 
-Make sure `KUBECONFIG` is set to point at your connected clusters:
+Set `KUBECONFIG` to point at your clusters:
 
 ```shell
-export KUBECONFIG=<paths>
+export KUBECONFIG=<kubeconfig0 path>:<kubeconfig1 path>
 ```
 
-Get an idea of what's failing:
+Show overview of, and diagnose issues with, each cluster:
 
 ```shell
+subctl show all
 subctl diagnose all
+```
+
+Diagnose common firewall issues between a pair of clusters:
+
+```shell
 subctl diagnose firewall inter-cluster <kubeconfig0 path> <kubeconfig1 path>
 ```
 
+{{% notice warning %}}
```

Comment:
Should this be a `warning` or a `note/info`?

Reply:
Thanks for the suggestion @sridhargaddam, `info` does look better to me, more middle-ground level of warning.